src/main/java/picard/sam/SamErrorMetric/CollectSamErrorMetrics.java

```diff
@@ -564,20 +566,13 @@ private static boolean checkLocus(final VCFFileReader vcfFileReader, final SamLo
         if (locusInfo != null) {
             try (final CloseableIterator<VariantContext> vcfIterator = vcfFileReader.query(locusInfo)) {
 
-                overlaps = true;
-
-                boolean allFiltered = true;
-
                 while (vcfIterator.hasNext()) {
-                    final VariantContext vcf = vcfIterator.next();
-                    if (vcf.isFiltered()) {
+                    final VariantContext variantContext = vcfIterator.next();
+                    if (variantContext.isFiltered()) {
```

Comment:
you could inline and skip the local variable, right?

Reply:
Yes.  Fixed.